RELEASE.md

```diff
@@ -9,6 +9,11 @@ Gradle plugin, which this repository contains.
 The credentials are stored in the same place as the credentials used to make
 a Checker Framework release, in a `gradle.properties` file.
 To gain access to the credentials, contact one of the maintainers privately.
+* Ensure that you have checked out the `master` branch of the project. Do
```

Comment:
I feel that these are in the wrong part of the file.  This part of the file is for one-time setup.
The two added sentences, however, are part of the release process and must be done for each release.

Reply:
Yes, I think you're right. I've moved them.